includes/class-sender.php

```diff
@@ -208,14 +208,15 @@ public static function send_webmentions( $post_id ) {
 		$urls = webmention_extract_urls( $post->post_content, $support_media_urls );
 
 		// filter links
-		$targets = apply_filters( 'webmention_links', $urls, $post_id );
-		$targets = array_unique( $targets );
-		$pung    = get_pung( $post );
```

Comment:
What is the problem with re-using `get_pung`? If we use something different it might result in duplicate pings. Not everyone who uses Webmentions has Pingbacks disabled.

Reply:
In the long term, I considered storing more about the ones that failed as an improvement of the retry system, but at the moment, I suppose I could revert part of this.

Reply:
@pfefferle Restored get_pung...but still have an independent webmention list. Can you have a look?